src/test_sband.py

```diff
@@ -0,0 +1,76 @@
+'''
```

Comment:
I was thinking of a separate directory for test scripts. There was a lot of conflicts in the import commands so I gave it up. But I think it's needed. What are your thoughts?

Reply:
yeah I agree, but we should maybe have a common test library so we don't repeat the `sendAndExpect` method for instance